tests/ci/cdk/cdk/ssm/general_test_run_ssm_document.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+---
+schemaVersion: '2.2'
+description: aws-lc:bmFrameworkEc2Benchmark
+mainSteps:
+  - action: aws:runShellScript
+    name: runShellScript
+    inputs:
+      timeoutSeconds: '7200'
+      runCommand:
+        - sudo -i
+        - export DEBIAN_FRONTEND=noninteractive
+        - export CPU_TYPE=$(dpkg --print-architecture)
+        # if we have a cpu type of x86, we want linux-x86
+        - if [ "${CPU_TYPE}" = amd64 ]; then export CPU_ARCH=linux-x86; export AWS_CLI_PREFIX=x86_; sudo sh -c "echo 1 > /sys/devices/system/cpu/intel_pstate/no_turbo"; fi
+        # if we have a cpu type of arm, we want linux-aarch
+        - if [ "${CPU_TYPE}" = arm64 ]; then export CPU_ARCH=linux-aarch; export AWS_CLI_PREFIX=aarch; fi
+        # install aws-cli
+        - apt-get -y install unzip
+        - curl "https://awscli.amazonaws.com/awscli-exe-linux-${AWS_CLI_PREFIX}64.zip" -o "awscliv2.zip"
+        - unzip awscliv2.zip
+        - ./aws/install
+        - git clone {GITHUB_REPO} aws-lc-pr
+        - cd aws-lc-pr
+        - git checkout {COMMIT_ID}
+        # install docker if its not already installed
+        - chmod +x ./tests/ci/benchmark_framework/install_docker.sh
+        - ./tests/ci/benchmark_framework/install_docker.sh
+        # log into docker and get needed docker image from ecr
+        - export ECR_REPO="{AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-${CPU_ARCH}"
+        - aws ecr get-login-password --region us-west-2 | docker login -u AWS --password-stdin "${ECR_REPO}"
+        - docker pull "${ECR_REPO}:amazonlinux-2023_clang-15x_sanitizer"
```

Comment:
Probably make the image name a variable, as it is repeated below in the exec_docker definition.